portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSet.path

```diff
@@ -160,7 +170,7 @@
 </tr>
 <tr>
 	<td>RESET_FILTERS</td>
-	<td>//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), 'Reset Filters')]</td>
+	<td>//*[contains(@class,'management-bar')]//*[@type='button' and contains(text(), '${reset_filter}')]</td>
```

Comment:
for variables within *.path files, our naming convention is to start with `key_`